src/Microsoft.EntityFrameworkCore.AutoHistory/AutoHistory.cs

```diff
@@ -25,7 +25,7 @@ public class AutoHistory
         /// Gets or sets the name of the table.
         /// </summary>
         /// <value>The name of the table.</value>
-        public string TableName { get; set; }
+        public string EntityName { get; set; }
```

Comment:
No, No, SHOULD add new property name `EntityName` rather than rename the `TableName`, rename or change it's behavior are breaking change, so we should add a new property to solved it.